prov/psm2/src/psmx2.h

```diff
@@ -955,6 +955,39 @@ void	psmx2_ns_add_local_name(int service, struct psmx2_ep_name *name);
 void	psmx2_ns_del_local_name(int service, struct psmx2_ep_name *name);
 void	*psmx2_ns_resolve_name(const char *server, int *service);
 
+static inline
```

Comment:
the following two functions can be removed if the utility code takes raw address instead of string (in deed it does, but the prototype creates a illusion that a string is expected).